packages/commonwealth/client/scripts/views/components/react_quill_editor/markdown_formatted_text.tsx

```diff
@@ -11,7 +11,7 @@ import 'components/react_quill/markdown_formatted_text.scss';
 import DOMPurify from 'dompurify';
 import { loadScript } from 'helpers';
 import { twitterLinkRegex } from 'helpers/constants';
-import { debounce } from 'lodash';
+import _ from 'lodash';
```

Comment:
Does importing functions directly from 'lodash' break builds?

Reply:
Yeah; lodash (at least the version we're using) doesn't export named imports in the module.